extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -369,6 +391,17 @@ angular.module('oppia').component('svgFilenameEditor', {
         return ctrl.diagramStatus === STATUS_SAVED;
       };
 
+      ctrl.customToSVG = function(toSVG, type, id) {
```

Comment:
Function names shoudl start with a verb. Please change this.

Reply:
Done.